README.md

```diff
@@ -59,20 +151,364 @@ of these challenges.
+  each day to achieve mastery. 👩‍💻 
+  + etc ... 
+  
+  > There are _many_ areas where people can have life goals. 
+  Eventually our app will encourage people 
+  to have goals defined for each area of a person's life. 
+  We will have an on-boarding questionnaire to guide them, 
+  a visualisation of "progress" toward each one of the goals 
+  and reminders to review them regularly. 
+  All features to be added to the backlog and worked on in due course! 
+  For now, we need the basic
+  [**Personas**](https://github.com/dwyl/app/issues/213)
+  for individuals using the App to guide our discussions.
++ **categorise** the **`items`** (_of **`text`**_) 
+they brain-dumped into meaningful categories using `tags`
+
+  > ***Ideally*** people should tie each `item` 
+  to a specific goal so that it's clear to them 
+  _why_ they need to do something or _why_ 
+  a particular `item` of information is _relevant_ to them. 
+  > ***However*** we aren't _ever_ going to _force_ 
+  any particular "system" on people.
+  If someone wants to use our app just 
+  to take random  `plaintext` notes that's 100% OK. 
+  We will write Blog posts (_product release notes_) 
+  for the features we build in the future 
+  that will _guide_ people toward having 
+  a more systematic approach. 
+  e.g: set goals, write everything down, 
+  tie all tasks to specific goals, etc. 
+  But until we _build_ those features into our App, 
+  we _must_ make it _useful_ in `plaintext` mode.
+
+
++ ***prioritise*** the `items` that need to be done first,
+those that can be differed or delegated.
+
+  > "_If You Don’t Prioritize Your Life, Someone Else Will_". 
+  ~ [Greg McKeown](https://hbr.org/2012/06/how-to-say-no-to-a-controlling), 
+  [Essentialism](https://youtu.be/euJc0UmSik8)
+
+
++ ***Use*** the App to stay "on track" 
+toward _personal_ goals 
+and get the most important things done.
+
+
+<br />
+
+While we feel the _individual_ experience of the App
+needs to be flawless as we want anyone 
+to be able to use the app in isolation,
+we feel the App becomes _exponentially_ more useful/powerful in _teams_.
+
+
+
+<br />
+
+### 2. Small _Team_ Experience -> [_Co-op Mode_](https://en.wikipedia.org/wiki/Cooperative_gameplay)
+ 
+
+We feel that our App will be _incredibly_ helpful
+to small "teams" of people who need to collaborate on getting things done.
+Small teams will not be limited to "work" colleagues.
+Rather we will have personas for families, groups of friends and other associations. 
+We will focus on this "team" use-case 
+as soon as we have the basic _individual_ UX working.
+
+> If you are keen on helping us with the Colab features,
+please take a look at our existing backlog: 
+https://github.com/dwyl/app/issues 
+If there is a feature you don't see and want to help us describe, 
+please open an issue!
+
+
+## 3. Big Company/Org Experience -> [_MMORPG_](https://en.wikipedia.org/wiki/Massively_multiplayer_online_role-playing_game)!
```

Comment:
Third point of the stages of the roadmap should have the same type of title to the two previous points
```suggestion
### 3. Big Company/Org Experience -> [_MMORPG_](https://en.wikipedia.org/wiki/Massively_multiplayer_online_role-playing_game)!
```